api/log_test.go

```diff
@@ -54,8 +65,12 @@ func TestLogWithCheckingPanic(t *testing.T) {
 					t.Errorf("got: %q, want: %q", got, tc.want)
 				}
 			}()
-			defer logWithCheckingPanic(f, inputStatusCode, inputError)
-			panic(tc.input)
+			statusCode := http.StatusOK
```

Comment:
nit: since we're going to override it later in every case, maybe not giving it a value here can make that more clear.
```suggestion
			var statusCode int
```

Reply:
🙇